### ISSUE TITLE
Mushe or Moses

### ISSUE THREAD

**Issue by davidamichelson** [srophe/srophe-app-data#190]
Any thoughts? I feel here like leaving it Mushe may be the best fit?

**Comment by davidamichelson** [comment]
example: http://wwwb.library.vanderbilt.edu/exist/apps/srophe/person/173

**Comment by nathangibson** [comment]
My opinion on headwords is to use the anglicized form **unless** that contradicts normal usage. 

Mushe bar Kipho (http://wwwb.library.vanderbilt.edu/exist/apps/srophe/person/173) would be a good example of normal usage ("Mushe") overriding the anglicized version ("Moses"), whereas I think "Moses of Nisibis" (http://wwwb.library.vanderbilt.edu/exist/apps/srophe/person/647) would be the opposite.

In other words, I would be in favor of using "Moses" in general, but making exceptions where needed.

**Comment by nathangibson** [comment]
The guiding principle here would be recognizability. An anglicized version is generally more recognizable unless the person is well-known, and thus may have another version.

**Comment by jsaintlauren** [comment]
I agree with Nathan here. 

(Sorry to be late in answering this questions.  I am at NAPS :)

Syriaca got a plug from Sarah Bond! )

**Comment by nathangibson** [comment]
Yay for the plug!

**Comment by davidamichelson** [comment]
But I don't like the result that in the browse all the Moses would not be together since some would be under Mushe 

Would users be expecting this?

**Comment by jsaintlauren** [comment]
I think as long as they are searchable in the big browse under both names that is fine.  I will make a note about this in the usage guide.
